k8s/cassandra/Makefile

```diff
@@ -63,3 +67,12 @@ app/build:: .build/cassandra/cassandra \
 	    "$(REGISTRY)/cassandra:$(TAG)"
 	docker push "$(REGISTRY)/cassandra:$(TAG)"
 	@touch "$@"
+
+.build/cassandra/tester:   .build/var/TESTER_IMAGE \
+                           $(shell find apptest -type f) \
+                           | .build/cassandra
+	$(call print_target, $@)
```

Comment:
I think you probably want to remove the space after `,`, as make is sensitive to all characters

Reply:
Good catch, thanks!